### PR TITLE
Bump to 3.0.3-1 and upgrade dependencies

### PR DESCRIPTION
**Pull request by carlonzo**

### CODE REVIEW

**Comment by carlonzo** [comment]
@blazsolar is CircleCI configured with this project? how can I check the status?

**Comment by blazsolar** [comment]
Hi, can you reopen this pull request. As I just enabled PR building on CircleCI. After that you should see CI status. 

Thanks

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-VERSION_NAME=3.0.0
-VERSION_CODE=3000000
+VERSION_NAME=3.0.3-1
+VERSION_CODE=3003001
```
Comment (blazsolar) [inline gradle.properties]: this should be `3000301`

**Review by blazsolar — CHANGES_REQUESTED** [review]
Please update the comment and also bump version in `README.md` file. After that I will merge this PR. Thanks

File: gradle.properties (hunk 1 above)
Comment (carlonzo) [reply gradle.properties]: thanks! I was not sure about that :)

**Comment by carlonzo** [comment]
@blazsolar done. thanks

**Review by blazsolar — APPROVED** [review]

**Comment by blazsolar** [comment]
Thanks for contributing, i'll make release tomorrow

**Pull request merged**